framework/src/main/java/org/checkerframework/common/wholeprograminference/SceneToStubWriter.java

```diff
@@ -54,7 +54,7 @@
  * <p>This class works by taking as input a scene-lib representation of a type augmented with
  * additional information, stored in javac's format (e.g. as TypeMirrors or Elements). The A*Wrapper
  * classes ({@link ASceneWrapper}, {@link AClassWrapper}, {@link AMethodWrapper}, and {@link
- * AFieldWrapper}) store this additional information. This class walks the scene-lib representation
+ * AField}) store this additional information. This class walks the scene-lib representation
```

Comment:
Changing this to `AField` doesn't make sense, because this comment's purpose is to explain the need for the *Wrapper classes. If you've added the contents of `AFieldWrapper` to `AField`, it obviates the need to mention `AField` here at all.